dinamo.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "activesupport", "~> 5.0.0"
-  spec.add_dependency "aws-sdk",       "~> 2.1.26"
+  spec.add_dependency "aws-sdk-dynamodb", "~> 1"
```

Comment:
Please specify minor version, it would be something like: `~> 1.2`. Btw, I'm not sure what we should also add `aws-sdk-core` as a dependency, the current version seems to depend on `~> 3`. Is that reasonable?

Reply:
Ok, specify the minor version.
No, it doesn't change. I also add `aws-sdk-core` to dependencies explicitly.